templates/Dockerfile-deb.tmpl

```diff
@@ -57,22 +57,25 @@ RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ %%release%%-pgdg main' >
         libxslt-dev \
         lintian \
         postgresql-server-dev-all \
-        wget \
+        # Add 12 package explicitly as *-all does not yet depend on 12 yet
+        postgresql-server-dev-12 \
+                wget \
     && rm -rf /var/lib/apt/lists/*
 
 # install jq to process JSON API responses
 RUN curl -sL https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 \
          -o /usr/bin/jq \
     && chmod +x /usr/bin/jq
 
-# install latest Citus release to get header files
+# DO NOT install latest Citus release to get header files
 RUN curl -s https://packagecloud.io/install/repositories/citusdata/community/script.deb.sh | bash \
-    && apt-get install -y postgresql-9.6-citus-8.0 postgresql-10-citus-8.0 postgresql-11-citus-8.0 \
+#    && apt-get install -y postgresql-9.6-citus-8.0 postgresql-10-citus-8.0 postgresql-11-citus-8.0 \
     && rm -rf /var/lib/apt/lists/*
 
+# FIXME: get the patches working again
```

Comment:
I've pushed a commit to fix this patch, so you should just revert the change here.

Reply:
How important is this patch ? I remember removing this patch line when I was creating test packages locally.

Reply:
It can speed things up. I will revert changes and update all the patches in relevant directories.